### DISCUSSION TITLE
Want to know about the repositories that we can contribute to improve our knowledge

### DISCUSSION THREAD

**Discussion by heheboii11** [Cyfrin/foundry-full-course-cu#2945] [General]
Hey @everyone, can you please suggest the git repo or any places we can contribute , same as we having the discussions over here.

**Comment by LordAdhaar** [comment]
Interesting, maybe find a project that you find interesting that is solving a probelm you like, and try to contrifbute to their repo

**Reply by EngrPips** [reply]
Good heads-up.

**Reply by heheboii11** [reply]
Thanks for the suggestion!!!
